db/schema.rb

```diff
@@ -0,0 +1,36 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_09_13_104950) do
```

Comment:
It's a good idea to create an index on the columns we will use to access a particular table, i.e. foreign key columns. 

Here are a couple additional resources on that matter:
* https://semaphoreci.com/blog/2017/05/09/faster-rails-is-your-database-properly-indexed.html
* https://sqlperformance.com/2012/11/t-sql-queries/benefits-indexing-foreign-keys.

Please look into how you can add indexes to an already created table by means of a migration.

[`REQUIRED`]